components/database.js

```diff
@@ -45,6 +45,25 @@ export default class Database {
     }
   }
 
+  static async getUser(uname) {
```

Comment:
```suggestion
  static async getUser(uname) {
    try {
      let user;
      await AsyncStorage.getItem("users").then((data) => {
        JSON.parse(data).forEach((element) => {
          if (element.username == uname){
            user = element;
          }
        });
      });
```

Another way instead of doing for loop you can run the iterator on the parsed array